tests/unit/test_rescue.py

```diff
@@ -83,36 +83,36 @@ def test_created_at_date_exists(rescue_sop_fx):
     """
     Verifies rescue.created_at datetime is set, and in the past.
     """
-    expected_time_differential = (datetime.now(tz=tzutc()) - rescue_sop_fx.created_at)
+    expected_time_differential = (pendulum.now() - rescue_sop_fx.created_at)
```

Comment:
[style] We should be consistent, is it pendulum.now() or pendulum.now(pendulum.tz.UTC). Let's try to also keep this within the tests
[style] Also for consistency: `pendulum.datetime()` or `pendulum.DateTime()`?

Reply:
For the first, see above.
for the latter, :thinking: im pretty sure the actual class is `DateTime` and `datetime` is an alias, to ease the transition / interop with stdlib

Reply:
Poking into documentation, it appears `datetime` is the public API component, `Datetime` is the class....

anywhere where im doing `pendulum.DateTime` is incorrect, and should actually be `pendulum.datetime`